spec/features/editing_tags_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+feature "Editing Tags", :tag_helper, type: :feature do
+  seed_tags
+
+  let(:admin) { create_admin_user }
+  let(:normal_user) { create_really_basic_user }
+  let(:user) { admin }
+  before(:each) { login_as(user, scope: :user) }
+  after(:each) { logout(:user) }
+
+  feature 'Admins can edit tag attributes' do
+    let(:tag) { Tag.find(1) } # oil
+    before(:each) { visit edit_tag_path(tag) }
+
+    it 'displays a form with the tag information' do
+      expect(page.status_code).to eq 200
+      expect(page).to have_current_path edit_tag_path(tag)
+      expect(page).to have_content "Editing Tag:"
+      expect(page).to have_content #{tag.name}"
+      expect(page).to have_selector 'form.edit_tag'
+      expect(page).to have_selector '#delete-this-tag'
+    end
+
+    scenario 'Admin can change the description of the tag' do
+      fill_in('Description', with: 'prefers profit over people')
```

Comment:
you're really killing it on the example text today! 🤖 👍 🤖

Reply:
lol.